nav2_graceful_motion_controller/include/nav2_graceful_motion_controller/path_handler.hpp

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2023 Alberto J. Tudela Roldán

```

Comment:
If this is taken from other Nav2 projects, it should use that project's header

Reply:
It's based on RPP path class but it's not 100% exactly. Should I the RPP header in that case?

Reply:
Yeah. If you copy+paste files, you must retain the old headers. If you substantially change it from there, then you can add your own copyright line in addition to the original, but only for substantive changes. What that means is somewhat subjective, but I'm also not really overly picky about it. Use your best judgement.